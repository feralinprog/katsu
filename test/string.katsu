use: {
    "core.builtin.misc"
    "core.sequence"
    "core.sequence.string"
}

# See https://en.wikipedia.org/wiki/UTF-8, "UTF-8 encoding example"
let: s = "WΒ위𐍅"
print: "code units:"
s code-units each: [ pretty-print: it ]
print: "code points:"
s code-points each: [ pretty-print: it ]
