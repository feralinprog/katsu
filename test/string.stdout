code units:
fixnum 87
fixnum 206
fixnum 146
fixnum 236
fixnum 156
fixnum 132
fixnum 240
fixnum 144
fixnum 141
fixnum 133
code points:
fixnum 87
fixnum 914
fixnum 50948
fixnum 66373
